extras/jenkins/AWS/Jenkinsfile

```diff
@@ -188,7 +188,7 @@ pipeline {
 
           sh '''
             $WORKSPACE/bin/destroy.sh
-            find . -mindepth 2 -maxdepth 6 -type f -name Pulumi.yaml -execdir pulumi stack rm marajenkaws${BUILD_NUMBER} --force --yes \\;
+            find . -mindepth 2 -maxdepth 6 -type f -name Pulumi.yaml -execdir $WORKSPACE/pulumi/python/venv/bin/pulumi stack rm marajenkaws${BUILD_NUMBER} --force --yes \;
```

Comment:
Just for my learning, what was the additional change here?  It appears now to be in line with the other workflows, but I wonder why it was different in the first place.

Reply:
Two reasons - first I wanted to keep them all consistent and second I wanted to make sure that we are always using the pulumi binary that we install in the venv. I've been using the jenkinsfiles as a template for other scripts / processes (like the UDF work) so I want to make sure that we're a bit tighter here.